spec/controllers/Admin/profile_controller_spec.rb

```diff
@@ -1,21 +1,281 @@
+require 'spec_helper'
```

Comment:
Isn't that the 'spec_helper' already required in the .rspec file? So we don't need that here?